### PR TITLE
Kind installation

### PR DESCRIPTION
**Pull request by ndeloof**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -6,7 +6,6 @@ aliases:
     - kube_1_13: &kube_1_13 "1.13.7"
     - kube_1_14: &kube_1_14 "1.14.6"
     - kube_1_15: &kube_1_15 "1.15.0"
-  kind_version: &kind_version "0.5.1"
```
Comment (chris-crone) [inline .circleci/config.yml]: Do we not want to keep this as a variable at the top of the file?

File: .circleci/config.yml [hunk 2]
```diff
@@ -28,12 +27,8 @@ aliases:
     - run:
         name: "Prepare Kind and run e2e tests"
         command: |
-          git clone https://github.com/kubernetes-sigs/kind.git /root/src/sigs.k8s.io/kind
-          cd /root/src/sigs.k8s.io/kind
-          git reset --hard 3907329b4ece1feab97d6281de23ea6e11812a1a
-          GO111MODULE=on go mod vendor
-          go install
-          cd -
+          curl -Lo /root/bin/kind --create-dirs https://github.com/kubernetes-sigs/kind/releases/download/v0.5.1/kind-$(uname)-amd64
```
Comment (chris-crone) [inline .circleci/config.yml]: 👍

File: .circleci/config.yml (hunk 1 above)
Comment (ndeloof) [reply .circleci/config.yml]: right, need to investigate passing variable in circle-ci. First time I tried it resulted in some unset value

Comment (ndeloof) [reply .circleci/config.yml]: I moved kind installation to Makefile as this can be useful as well for contributors

**Comment by silvin-lubecki** [comment]
Tests are failing:
```
make: kind: Command not found
make: *** [docker.Makefile:97: create-kind-cluster] Error 127
Exited with code 2
```

**Pull request merged**